### ISSUE TITLE
Error with the start.replace function

### ISSUE THREAD

**Issue by frankmeeuwsen** [dariusk/roof-slapping-bot#2]
I tried your bot (it's fun!) with my testaccount. I ran into this error after a while:

`TypeError: Cannot read property 'replace' of undefined
    at Promise (/roof-slapping-bot/index.js:15:19)`

I think it has to do with the regex but not quite sure how to fix it (yet).
I want to try to make a Dutch version of your bot with another JSON from conceptnet.

**Comment by dariusk** [comment]
Yes, sometimes there is no `surfaceStart`. The solution would be to either test for the variable's presence and pull a new random variable if it doesn't exist, or remove entries without the variable from the data file. Fortunately it's a bot so it doesn't need to run every time. Failure just means it won't post to Twitter this time around and will try again next time.
